providers/catalog.go

```diff
@@ -1215,4 +1216,29 @@ var catalog = CatalogType{ // nolint:gochecknoglobals
 			Write:     false,
 		},
 	},
+
+	// Zuora Configuration
+	Zuora: {
+		AuthType: Oauth2,
+		BaseURL:  "https://rest.test.zuora.com",
```

Comment:
I'm guessing this came from here:

https://developer.zuora.com/api-references/api/overview/#section/Base-URL

I know that this ticket is about getting the sandbox working. So in that sense this is fine. But we also need to support the production URLs. To help with that, I'd suggest changing this from a hard-coded domain to something we can substitute.

```suggestion
		BaseURL:  "https://{{.subdomain}}.zuora.com",
```

Reply:
updated